self-core-impl/src/main/java/com/selfxdsd/core/GitlabIssueLabels.java

```diff
@@ -94,9 +89,18 @@
 
     @Override
     public boolean add(final String... names) {
+        final GitlabRepoLabels repoLabels = new GitlabRepoLabels(
+            URI.create(
+                this.uri.toString()
+                    .replaceAll("/issues/[0-9]+", "/labels")
+            ),
+            this.resources
+        );
+        repoLabels.add(names);
         final boolean added;
         final String labels = Arrays.stream(names)
             .collect(Collectors.joining(","));
+        System.out.println("Issue Labels: " + this.uri.toString());
```

Comment:
@amihaiemil Is this intended or just forgotten, because it's already logged?

Reply:
@fellahi-ali it's forgotten there, I'll remove it now. Thanks!